src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,172 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final double HALF = 0.5;
+    private static final int ONE = 1;
```

Comment:
Not sure if there is some sense in this constant

Reply:
agree (HALF as well) Or you can think and give better names to constants